vertica/metadata.csv

```diff
@@ -34,6 +34,7 @@ vertica.node.disk.used.ros,gauge,,byte,,[Only versions <= 10] The ROS disk space
 vertica.node.disk.used.wos,gauge,,byte,,[Only versions <= 10] The WOS disk space used by nodes in bytes.,-1,vertica,,
 vertica.node.down,gauge,,node,,The number of nodes that are currently down.,-1,vertica,,
 vertica.node.resource_requests,gauge,,resource,,"The cumulative number of requests for threads, file handles, and memory (in kilobytes).",0,vertica,,
+vertica.node.ros.containers,gauge,,resource,,The number of ROS containers in nodes.,-1,vertica,,
```

Comment:
Are there any other metrics that are deprecated or no longer available in Vertica 11/12 that should be documented here? Or new metrics that aren't available in older vertica versions?

Reply:
These metrics are available in all versions (pre and post Vertica 11). I added them for completeness and consistency, since it was the only one on its group of metrics that we were generating only _per projection_.